webdriver-spec.html

```diff
@@ -8474,196 +8535,69 @@
-   </ol>
-  </ol>
-
- <li><p>If <var>element</var> is a <a><code>text</code> node</a>,
-  return true.
-
- <li><p>If it has equal to or more than one direct descendant elements:
-
-  <ol>
-   <li><p>Let <var>visible children</var> be a boolean initially set to false.
-
-   <li><p>For each direct descendant element <var>child</var>:
-
-   <ol>
-    <li><p>Let <var>rectangle</var> be <var>child</var>’s <a>bounding rectangle</a>.
-
-    <li><p>If the value of the <code>height</code> property of <var>rectangle</var> is
-     greater than zero <a>CSS reference pixels</a>,
-     and the value of the <code>width</code> property of <var>rectangle</var>
-     is greater than zero <a>CSS reference pixels</a>:
-
-     <ol>
-      <li><p>Set <var>visible children</var> to <var>visible children</var> bitwise OR true.
-     </ol>
-   </ol>
-  </ol>
-
- <li><p>For each ancestral element <var>parent</var>, in <a>tree order</a>:
-
-  <ol>
-   <li><p>Apply the <a>element displayed</a> algorithm to <var>parent</var>.
-
-   <li><p>If the return value is false, abort these steps and return that value.
-
-   <li><p>If <var>parent</var> is a block element box and the
-    computed values of either <code>overflow-x</code>
-    or <code>overflow-y</code> is "hidden":
-
-   <ol>
-     <li><p>Let <var>parent dimensions</var> be the <a>rectangle</a>
-     that is the first element of the <a><code>DOMRect</code></a>
-     <a>sequence</a> returned by calling <a>getClientRects</a> on <var>parent</var>.
-
-    <li><p>Let <var>element dimensions</var> be the <a>rectangle</a>
-     that is the first element of the <code>DOMRect</code>
-     <a>sequence</a> returned by calling <a>getClientRects</a> on <var>element</var>.
-
-    <li><p>Let <var>parent style</var> be the computed style of <var>parent</var>.
-
-    <li><p>Return false if any the following conditions evaluate to false:
-
-    <ul>
-     <li><p><var>element dimension</var>'s <code>top</code>
-      is less than (<var>parent dimension</var>'s <code>bottom</code>
-      − the <a>normalized style pixel float value</a>
-      of <var>parent style</var>'s <code>borderBottomWidth</code>).
-
-     <li><p><var>element dimension</var>'s <code>bottom</code>
-      is less than (<var>parent dimension</var>'s <code>top</code>
-      − the <a>normalized style pixel float value</a>
-      of <var>parent style</var>'s <code>borderTopWidth</code>).
-
-     <li><p><var>element dimension</var>'s <code>left</code>
-      is less than (<var>parent dimension</var>'s <code>right</code>
-      − the <a>normalized style pixel float value</a>
-      of <var>parent style</var>'s <code>borderRightWidth</code> casted as a float).
-
-     <li><p><var>element dimension</var>'s <code>right</code>
-      is less than (<var>parent dimension</var>'s <code>left</code>
-      − the <a>normalized style pixel float value</a>
-      of <var>parent style</var>'s <code>borderLeftWidth</code> casted as a float).
-   </ul>
+<div class=note>
```

Comment:
I can’t find any references to **element displayed** in the document. Everything else is defined in terms of pointer-interactability.